.circleci/config.yml

```diff
@@ -361,7 +361,7 @@ jobs:
     working_directory: ~/experimenter
     environment:
       FIREFOX_VERSION: nimbus-firefox-release
-      PYTEST_ARGS: -k 'not (FOCUS_IOS or DESKTOP or FENIX or FOCUS_ANDROID or IOS)'  -m demo_app --reruns 1
+      PYTEST_ARGS: -k DEMO_APP  -m demo_app --reruns 1
     steps:
```

Comment:
I tried this, but for some reason, it was running for all, that's why I explicitly said not these apps

Reply:
Yeah I'm updating the names so that we should be able to just specify the one app.